hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/RepositoryApplicationConfiguration.java

```diff
@@ -854,4 +854,10 @@ RolloutScheduler rolloutScheduler(final TenantAware tenantAware, final SystemMan
             final RolloutManagement rolloutManagement, final SystemSecurityContext systemSecurityContext) {
         return new RolloutScheduler(systemManagement, rolloutManagement, systemSecurityContext);
     }
+
+    @Bean
+    ActionStatusUpdateHandlerService actionStatusHandlerService(final ControllerManagement controllerManagement,
+            final EntityFactory entityFactory) {
+        return new ActionStatusUpdateHandlerService(controllerManagement, entityFactory);
```

Comment:
this results in a compilation error, missing third parameter
maybe somehow lost while committing? :)
```
    @Bean
    ActionStatusUpdateHandlerService actionStatusHandlerService(final ControllerManagement controllerManagement,
            final EntityFactory entityFactory, final SystemSecurityContext systemSecurityContext) {
        return new ActionStatusUpdateHandlerService(controllerManagement, entityFactory, systemSecurityContext);
    }
```

Reply:
Thats true. I had missed to commit this file.